schedule/yam/create_hdd_module_combinations_sles12sp5_default_pattern.yaml

```diff
@@ -0,0 +1,19 @@
+name: create_hdd_module_combinations_sles12sp5_default_pattern
+description:    >
+  Create support images with module combinations
+  on default patterns on sles12sp5.
+conditional_schedule:
+  svirt_upload_assets:
+    ARCH:
+      s390x:
+        - shutdown/svirt_upload_assets
+schedule:
+  - installation/bootloader_start
+  - boot/boot_to_desktop
+  - update/patch_sle
+  - console/scc_deregistration
+  - console/force_scheduled_tasks
```

Comment:
I think maybe we can skip this test to avoid the risk of failure of this module, and it won't affect we create the qcow. WDYT?

Reply:
yes, I think so.